community/zig/build

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -e
+
+mkdir -p build
+cd build
+
+cmake \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    ..
+
+make DESTDIR="$1" install
```

Comment:
https://k1sslinux.org/wiki/kiss/style-guide#0800

```
export DESTDIR="$1"

cmake -B build \
    -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_INSTALL_PREFIX=/usr 

cmake --build   build
cmake --install build
```

Reply:
Swap the `PREFIX` and `BUILD_TYPE` :)